CreateSetupPackageLayout.proj

```diff
@@ -4,74 +4,43 @@
   <Import Project="$(MicroBuildDir)MicroBuild.Core.props" />
 
   <PropertyGroup>
+      <X86OutputPath>$(BaseOutputPath)X86\$(OS)\$(Configuration)\</X86OutputPath>
+      <X64OutputPath>$(BaseOutputPath)X64\$(OS)\$(Configuration)\</X64OutputPath>
       <SetupPackageRoot>$(BinDir)Setup\</SetupPackageRoot>
       <SetupPackageLayoutDir>$(SetupPackageRoot)layout\</SetupPackageLayoutDir>
       <SetupPackageOutputFile>$(SetupPackageRoot)MSBuild.fopx</SetupPackageOutputFile>
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- Commenting out files that aren't produced by the open source build
-    (When comparing against VS2015 installed to Program Files\MSBuild\14.0\Bin) -->
+    <BinX86Output Include="$(X86OutputPath)Microsoft.Build.dll" />
```

Comment:
The redundancy here is crying out for a "common" item + transforms. But it'll wait.